resources/assets/uad_lists.json

```
@@ -37664,9 +37724,219 @@
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.oplus.gesture",
+    "list": "Oem",
+    "description": "Always-on screen gestures.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.android.overlay.gmsconfig.common",
+    "list": "Oem",
+    "description": "Overlay to vendor required apps?\nTheres google apps and some not important oppo apps on the lists, not needed.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.android.launcher",
+    "list": "Oem",
+    "description": "OPPO Launcher",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.oplus.nrMode",
+    "list": "Oem",
+    "description": "OplusNrModeControl\nRequired for Smart 5G.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.oplus.qualityprotect",
+    "list": "Oem",
+    "description": "QualityProtect\nLogs about system usage, some apps, not useful.",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
     "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.virtualcomm",
+    "list": "Oem",
+    "description": "It's probably virtual communications.\nAlso about these it's in the code.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.safecenter",
+    "list": "Oem",
+    "description": "Security Center\nIt breaks 'display over other apps' permission!",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.oplus.screenshot",
+    "list": "Oem",
+    "description": "Screenshot\nNeeded for screenshots. Very useful app.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.google.android.connectivity.resources.overlay.oplus",
+    "list": "Oem",
+    "description": "Needed for network, may cause bootloop.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.oplus.subsys",
+    "list": "Oem",
+    "description": "Its app for virtual communication. Depends on 'com.oplus.virtualcomm'.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.oplus.uxdesign",
+    "list": "Oem",
+    "description": "Wallpapers & style\nNeeded for changing wallpaper.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.oplus.viewtalk",
+    "list": "Oem",
+    "description": "ViewTalk\nApplication for recognizing text in an image and reading it out loud.\nDid not figure out how it works.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
Some people may want to use it, so I think `Advanced` should make more sense.